create-publication-from-doi/src/main/java/no/unit/nva/doi/fetch/MainHandler.java

```diff
@@ -57,13 +62,18 @@
     private final transient BareProxyClient bareProxyClient;
     private final transient String publicationApiHost;
     private final transient String publicationApiScheme;
+    private final transient MetadataService metadataService;
 
     private static final Logger logger = LoggerFactory.getLogger(MainHandler.class);
 
+    /**
+     * Default constructor for MainHandler.
+     */
     @JacocoGenerated
     public MainHandler() {
         this(JsonUtils.objectMapper, new PublicationConverter(), new DoiTransformService(),
```

Comment:
After the PR is merged, can you remove the objectMapper parameter? We use only one value here.